src/components/cast/cast.js

```diff
@@ -15,12 +15,15 @@ const mapStateToProps = state => ({
   isCastAvailable: state.engine.isCastAvailable
 });
 
+// TODO Sakal - Oren, the name in the constructor is incorrect (should have been CastControl)
```

Comment:
1. current controls with Control ending - remove it
```
class CastControl extends BaseComponent
```
===>
```
class Cast extends BaseComponent
```

2. change all internal indices
3. in components index.js fix for BW compatibility for external use, i.e.
```
export {CastControl} from './cast';
```
===>
```
export {Cast, CastControl as PlayPauseControl} from './cast';
```